BETA_TESTING.md

```diff
@@ -1,47 +1,66 @@
-# Beta testing
+# Build DevTools
 
-This page describes the fastest way to test the latest version of DevTools with all experemental features enabled. Do not mix this setup with development environment.
+This page describes the fastest way to build DevTools with the goal to use it. Do not mix this setup with development environment. If you want to make code changes, follow [contributing guidance](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md).
 
-If you want to make code changes, follow [contributing guidance](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md).
+You may want to build DevTools to:
 
-The steps were tested for Mac. They may require adjustments for other platforms.
+1. Try experimental features
+
+2. Run desktop version, instead of Web version, to get get rid of browser memory limit. For example,
+to be able to analyze heap snapshots of more complicated applications.
+
+The steps were tested for Mac and may require adjustments for other platforms. Contributions,
+that make the steps more platform agnostic, are welcome.
 
 ## Prerequisites
 
 1. [Configure](https://docs.flutter.dev/get-started/install) Dart or Flutter.
 
-## Setup and start
+## Setup DevTools
```

Comment:
`## Setup DevTools (first time only)`

and then remove "if it is initial setup"